packages/select/src/FilterMultiSelect/components/ListBoxSection/ListBoxSection.tsx

```diff
@@ -3,41 +3,57 @@ import { v4 } from "uuid"
 import { VisuallyHidden } from "@kaizen/a11y"
 import { MultiSelectItem } from "../../../types"
 import styles from "./ListBoxSection.module.scss"
-export interface ListBoxSectionProps {
-  items: MultiSelectItem[]
+
+type SectionNameProps = {
   /**
    * Becomes an aria-label on the section, informing
    * unsighted users
    */
   sectionName: string
-  /** If provided, will override the aria-label for this group */
-  sectionHeader?: ReactNode
-  children: (item: MultiSelectItem) => React.ReactNode
 }
 
+type SectionHeaderProps = {
+  /**
+   * Becomes an aria-label on the section, informing
+   * unsighted users
+   */
+  sectionName?: string
+  /**
+   * Can be used for a visual title of the ListBoxSection or to provide addition information in a React node.
+   * If this is the same title as sectionName, you should only pass in a sectionHeader to avoid duplicate descriptions.
+   */
+  sectionHeader: ReactNode
```

Comment:
Do we really want to have `ReactNode` instead of `string`?

Reply:
It's so they can provide a InlineNotification